jicofo-selector/src/main/kotlin/org/jitsi/jicofo/bridge/Bridge.kt

```diff
@@ -58,6 +58,12 @@ class Bridge @JvmOverloads internal constructor(
         clock
     )
 
+    private val endpointRequestRestartRate = RateTracker(
+        config.endpointRestartRequestInterval,
```

Comment:
The val is "request restart" but the config is "restart request". Is there a difference between these two, or should they be the same?